clarity_ext/cli.py

```diff
@@ -82,20 +89,33 @@ def extension(module, mode, args, cache):
             key_values = (argument.split("=") for argument in separated)
             args = [{key: value for key, value in key_values}]
 
-        if mode == extension_svc.RUN_MODE_FREEZE:
+        validate_against_frozen = True # Indicates a run that should ignore the frozen directory
+        if mode == "test-fresh":
```

Comment:
Yes, no doubt. I'll fix it in the PR following this one, since it's a small fix.